multiple pages: rephrase without adjectives (#3863)
tldr-pages/tldr: pages/common/pass.md

# pass

> > A tool to store and read passwords or other sensitive data easily.

Reviewer: Perhaps this will read better?
```suggestion
> Tool for storing and reading passwords or other sensitive data.
```

Author: nice
